synapse/tools/genpkg.py

```diff
@@ -44,6 +44,23 @@ def loadOpticFiles(pkgdef, path):
                     'file': base64.b64encode(fd.read()).decode(),
                 }
 
+def loadOpticWorkflows(pkgdef, path):
+
+    wdefs = pkgdef['optic']['workflows']
+
+    for root, dirs, files in os.walk(path):
+
+        for name in files:
+
+            if name.startswith('.'):  # pragma: no cover
```

Comment:
is there a case where we have `.yaml` files in a workflows dir that starts with a `.` ?

Reply:
we could just skip all files that are not named `.yaml` instead?

Reply:
We could do something simple like `^([a-zA-Z0-9_-]+)\.yaml$` and log a warning for non-matching filenames

Reply:
I think sounds reasonable for now. It's not overly restrictive from future expansion.

Reply:
updated